### DISCUSSION TITLE
Bootstrap update on default theme

### DISCUSSION THREAD

**Discussion by fablaser** [dnnsoftware/Dnn.Platform#6177] [Ideas]
Hello, 
does anyone knows if it is planned an upgrade of bootstrap on Xcillion theme? It would be very useful.

Thanks!

**Comment by Timo-Breumelhof** [comment]
AFAIK there are currently no plans for or volunteers working on this.

**Comment by Timo-Breumelhof** [comment]
See also: https://github.com/dnnsoftware/Dnn.Platform/issues/2559

**Comment by david-poindexter** [comment]
We unfortunately cannot upgrade Bootstrap on the Xcillion theme, as it would be a breaking change for many websites.  For this reason, we are moving to a new theme for DNN 10 that is no longer framework dependent and is more maintainable moving forward.  😉  Live and learn.  😁👍🏼

**Comment by wbonekamp** [comment]
Great! When can we expect DNN 10 with the new theme? is there a release roadmap

**Reply by david-poindexter** [reply]
The best answer I can provide for roadmap is to check out the [10.0.0 milestone](https://github.com/dnnsoftware/Dnn.Platform/milestone/24).  You'll see many closed items and several remaining open items.  DNN Platform, as you know, is an open-source initiative and contributions are encouraged to help move the solution forward.  

The new theme, Aperture, is already completed (with the exception of a few documented minor items to polish things up).  There have been several presentations about it, but here is a [good intro to it](https://www.youtube.com/watch?v=xptWnadbgdQ&t=2700s).  That link starts at the 45 min mark of the video, but you may also be interested in the other parts of the video, as it touches on all areas of focus for DNN 10.
